source/eventcore/drivers/posix/sockets.d

```diff
@@ -205,6 +205,7 @@ final class PosixEventDriverSockets(Loop : PosixEventLoop) : EventDriverSockets
 			auto cb = connectCallback;
 			connectCallback = null;
 			if (cb) cb(cast(StreamSocketFD)sock, ConnectStatus.refused);
+			releaseRef(cast(StreamSocketFD)sock);
```

Comment:
will this also fix some other leaked sockets?

Reply:
This somehow slipped in. I'm 80% sure that this is an invalid change that just lingered in my working copy, but I'll check.